README.md

```diff
@@ -1,5 +1,7 @@
 # `supabase-js`
 
+[![Coverage Status](https://coveralls.io/repos/github/supabase/supabase-js/badge.svg?branch=master)](https://coveralls.io/github/supabase/supabase-js?branch=master)
```

Comment:
I have added coverage badge, remove if you don’t want to have it

Reply:
💯 thanks!